v2.0/getting-started/kubernetes/installation/hosted/k8s-backend/calico-daemonset.yaml

```diff
@@ -0,0 +1,99 @@
+# This manifest installs the calico/node container, as well
```

Comment:
The differences required for the add-on manager are that the two are split into separate files and that the "cluster-service" labels are included, right?

I think we might be able to simplify this a lot by moving the `cni_network_config` into the DaemonSet rather than through a ConfigMap, thus combining into a single object, and then documenting that for use with the add-on manager, make sure you've specified the correct set of labels.  

WDYT?

I'm not entirely convinced, I'm just trying to see what we can do to slow down the proliferation of self-hosted manifests which are likely to get out of sync with each other :)